tests/README.md

```diff
@@ -0,0 +1,7 @@
+# Test vectors
+
+# rsapss
+
+The test vectors are from [Project Wycheproof](https://github.com/google/wycheproof/tree/2196000605e45d91097147c9c71f26b72af58003).
+
+*Note:* SHA-1 is not supported in Hacl and thus not tested.
```

Comment:
Can you file a follow up to add the other readme content here?

Reply:
See https://github.com/cryspen/hacl-packages/issues/98.